### ISSUE TITLE
first run of `multirust update` produces errors

### ISSUE THREAD

**Issue by durka** [brson/multirust#70]
I installed multirust (using @spastorino's hopefully-soon-to-be-merged Homebrew formula) and ran `multirust update` straightaway. This happened for all three channels:

```
multirust: nightly revision:

rustc 1.2.0-nightly (b77d60adb 2015-05-29) (built 2015-05-29)
Process didn't exit successfully: `rustc -vV` (exit code: 1)
--- stderr
multirust: no default toolchain configured
/usr/local/bin/rustc: line 107: return: can only `return' from a function or sourced script
multirustproxy: assert_nz toolchain is empty
```

Running `multirust default stable` and then `multirust update` again seems to be the solution.

**Comment by xitep** [comment]
btw: the "can only `return' from ..." error should have been fixed by pull request 61.

**Comment by brson** [comment]
It does seem to be the same bug as https://github.com/mozilla/rust/pull/61. Can you confirm @durka?

**Comment by durka** [comment]
I guess you meant this repo's #61, haha. But yes, this bug seems to be fixed by installing HEAD as opposed to the 0.0.6 tag.
